.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,170 @@
+name: "Bug Report"
+description: "File a bug report to help us improve Nightwatch."
+body:
+  - type: "markdown"
+    attributes:
+      value: |
+        Thanks for creating an issue 😄!
+
+        Please search open/closed issues before submitting. Someone
+        might have asked the same thing before 😉!
+
+        We're all volunteers here, so help us help you by taking the time to fill out this template accurately. ❤️
+  - type: "input"
+    id: "description"
+    attributes:
+      label: "Description of the bug/issue"
+      description: "A brief description of the issue, how to reproduce it, and the expected behavior."
+      placeholder: |
+        When I ____, I expected ____ to happen but ____ happened instead.
+    validations:
+      required: true
```

Comment:
Now, we can make any input as a `required field`, and users will only be able to create an issue once they had added all the information.